import { Request, Response, NextFunction } from "express";

export function exceptionHandler(
  err: any,
  req: Request,
  res: Response,
  next: NextFunction
) {
  console.error("‚ùå Exception caught:", err);

  res.status(err.status || 500).json({
    success: false,
    message: err.message || "Internal Server Error",
  });
}
